Traceback (most recent call last):
  File "ungrib_surf.py", line 96, in <module>
    un.untar_the_files_prev(trs, job_path, job_row=pre_row)
  File "/homeappl/home/aliagadi/saltena_2018/wrf_management/wrf_management/ungrib.py", line 125, in untar_the_files_prev
    tf.extract(m, untar_path)
  File "/homeappl/home/aliagadi/appl_taito/miniconda3/envs/b36/lib/python3.6/tarfile.py", line 2052, in extract
    numeric_owner=numeric_owner)
  File "/homeappl/home/aliagadi/appl_taito/miniconda3/envs/b36/lib/python3.6/tarfile.py", line 2114, in _extract_member
    os.makedirs(upperdirs)
  File "/homeappl/home/aliagadi/appl_taito/miniconda3/envs/b36/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: '/wrk/aliagadi/DONOTREMOVE/wrf_management_data/runs/run_2019_02_20/2018_03_09/ungrib_surf/untar/2018_03_08'
